experimental/Schemes/src/elliptic_surface.jl

```diff
@@ -1368,7 +1383,8 @@ function linear_system(X::EllipticSurface, P::EllipticCurvePoint, k::Int64)
 
     I = saturated_ideal(defining_ideal(U))
     IP = ideal([x*xd(t)-xn(t),y*yd(t)-yn(t)])
-    issubset(I, IP) || error("P does not define a point on the Weierstrasschart")
+    # Test is disabled for the moment; should eventually be put behind @check!
```

Comment:
See the comment. This was another bottleneck killing us. So we should allow to switch it off.

Reply:
There could always be a bad choice of monomial ordering somewhere in the background -- or some ignored caching of groebner bases.

Reply:
alright ... it is just trivial over `k(t)[x,y]` but the groebner basis does not see this and works over `k[x,y,t]` ... then degrevlex is a terrible choice. O.k. lets just disable this test.